### DISCUSSION TITLE
Error when linking AWS Domain to Vercel Application

### DISCUSSION THREAD

**Discussion by Itz-MrJ** [vercel/community#2804] [Help] [answered]
### Summary

I am trying to link an AWS Domain to a vercel application. I am able to link the `https://www.something.com/` with cname records but I am not able to link the `https://something.com` with A records nor ns records. Please help me out on how I can route the `https://something.com/`. Image is attached to this thread.

### Steps to Reproduce

[Picture](https://media.discordapp.net/attachments/1116440889049501726/1116444702313558066/image.png)

**Comment by smaeda-ks** [comment]
@Itz-MrJ I can't find your `A` record on your domain `plxxxxxxxxx.com`. Did you add that in your Route53 setting?

**Reply by Itz-MrJ** [reply]
Yes I have. 
![Screenshot_2023-06-09-16-33-22-48_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/vercel/community/assets/68953408/efe381fe-4b9e-4b3b-9678-c6e73b150105)

I have added all the 3 records -
Cname 
Ns
A

EDIT: Been like 24hours since I've added those records

**Reply by Itz-MrJ** [reply]
@smaeda-ks Could you please help me out real quick? Sorry to hurry things up just that got deadlines to meet

**Reply by smaeda-ks** [reply, marked as answer]
@Itz-MrJ `Record name` should be an empty value.

**Reply by Itz-MrJ** [reply]
Worked thanks.
Maybe change it in the docs aswell to leave the record name empty instead of an `@`

**Comment by Itz-MrJ** [comment]
Guys can anybody help me real quick please?
